samples/react-sample-app/src/components/LogoutRequestDenied.tsx

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Let's follow a unified naming convention for file names.
`kebab-case` or `PascalCase`

Reply:
Resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17